### PR TITLE
Improvement/yocto

### PR DESCRIPTION
**Pull request by dkomljenovic2502**

### CODE REVIEW

File: conf/layer.conf
Comment (tronical) [inline conf/layer.conf:19]: Ok, so this makes the dependency explicit. I thought it would be better to keep this generic and also work with poky once those recipes are updated. But maybe this a better "out of the box" experience.

**Review by tronical — APPROVED** [review]

File: conf/layer.conf
Comment (dkomljenovic2502) [reply conf/layer.conf:19]: it is easier to track... as you have full control over what is needed, so when you stop needing it, it can be deleted from one place

**Comment by dkomljenovic2502** [comment]
btw i see it approved, but i can't merge it, as i don't have write access

**Comment by tronical** [comment]
Yes, as I said in the email: I was going to wait until you report build success before merging it :)

**Comment by dkomljenovic2502** [comment]
> Yes, as I said in the email: I was going to wait until you report build success before merging it :)

it passed!

**Pull request merged**
